svgdigitizer.py

```diff
@@ -18,8 +24,13 @@ def __init__(self, filename, xlabel=None, ylabel=None):
         
         self.ref_points, self.real_points = self.get_points()
         
-        self.xtrafo = self.get_trafo([self.ref_points['x1'][0], self.ref_points['x2'][0]], [self.real_points['x1'], self.real_points['x2']])
-        self.ytrafo = self.get_trafo([self.ref_points['y1'][1], self.ref_points['y2'][1]], [self.real_points['y1'], self.real_points['y2']])
+        self.scalebars = self.get_scalebars()
+        self.scaling_factors = self.get_scaling_factors()
```

Comment:
Maybe it's easier to turn these two into properties instead? I.e., write

```
@cached_property
def scalebars(self):
    ...
```

Reply:
We might consider doing this for most of the methods actually, so we get rid of state in the digitizer.

Reply:
I agree.